MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -90,7 +90,7 @@ open class BaseInstructionsBannerView: UIControl {
         }
     }
     
-    func set(_ instruction: VisualInstructionBanner?) {
+    public func set(_ instruction: VisualInstructionBanner?) {
```

Comment:
We should provide a good changelog note about how `update(for:)` has been renamed to `set(_:)` and should only be used when the banner should be updated. Previously, calling `update(for:)` could return really for you if an update was not necessary. This is not the case now and should be used sparingly to avoid perf hits.

Reply:
Nope, the `update(for:)` has been changed to `updateDistance(for:)`

Reply:
@bsudekum ☝️

Reply:
Right, but it's the idea I'm getting at. We need to tell developers how and when they need to update their top banner.

Reply:
That is going to be added in the changelog as a new API update.

Reply:
@vincethecoder yes, this is what I'm saying, `We should provide a good changelog note...`.